website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
+Sincerely yours,
+
+The OSF Robots
```

Comment:
We should ask community to write some documentation about what it means to retract a registration, and then provide a link to that in each email.  [Can be done following initial release if needed.]